### ISSUE TITLE
Fix torch Dropout sampling

### ISSUE THREAD

**Issue by MauriceFromm** [pwhofman/probly#500]
`nn.Dropout` in train mode draws an independent mask per sample in the batch. When the representer passes all grid points as a single batch each point gets a different random sub-network in every forward pass. This resulted in a "dotty" plot, while working on #492.
`DropConnectLinear` draws one mask per forward pass, which resulted in a smoother (not "dotty") plot.
